client/src/scenes/Game.ts

```diff
@@ -76,32 +80,52 @@ export default class Game extends Phaser.Scene {
     this.myPlayer = this.add.myPlayer(705, 500, 'adam', this.network.mySessionId)
     this.playerSelector = new PlayerSelector(this, 0, 0, 16, 16)
 
-    // import item objects (currently chairs) from Tiled map to Phaser
-    this.items = this.physics.add.staticGroup({ classType: Item })
+    // import chair objects from Tiled map to Phaser
+    const chairs = this.physics.add.staticGroup({ classType: Chair })
     const chairLayer = this.map.getObjectLayer('Chair')
     chairLayer.objects.forEach((chairObj) => {
-      const item = this.addObjectFromTiled(this.items, chairObj, 'chairs', 'chair') as Item
+      const item = this.addObjectFromTiled(chairs, chairObj, 'chairs', 'chair') as Chair
       // custom properties[0] is the object direction specified in Tiled
       item.itemDirection = chairObj.properties[0].value
     })
 
+    // import computers objects from Tiled map to Phaser
+    const computers = this.physics.add.staticGroup({ classType: Computer })
     const computerLayer = this.map.getObjectLayer('Computer')
-    var counter = 0
-    computerLayer.objects.forEach((Obj) => {
-      const item = this.addObjectFromTiled(this.items, Obj, 'computers', 'computer') as Item
+    let counter = 0
+    computerLayer.objects.forEach((obj) => {
+      const item = this.addObjectFromTiled(computers, obj, 'computers', 'computer') as Computer
       item.setDepth(item.y + item.height * 0.27)
       const id = String(counter)
       item.id = id
       this.computerMap.set(id, item)
       ++counter
     })
 
+    // import whiteboards objects from Tiled map to Phaser
+    const whiteboards = this.physics.add.staticGroup({ classType: Whiteboard })
+    const whiteboardLayer = this.map.getObjectLayer('Whiteboard')
+    counter = 0
+    whiteboardLayer.objects.forEach((obj) => {
+      const item = this.addObjectFromTiled(
+        whiteboards,
+        obj,
+        'whiteboards',
+        'whiteboard'
+      ) as Whiteboard
+      const id = String(counter)
+      item.id = id
+      this.whiteboardMap.set(id, item)
+      ++counter
```

Comment:
`forEach` has the index in second argument
```ts
whiteboardLayer.objects.forEach((obj, i) => {
  item.id = `${i}`;
})
```

Reply:
Done!